wdfn-server/waterdata/templates/base.html

```diff
@@ -9,10 +9,14 @@
 {% endblock %}
 
 {% block body %}
-    {% include 'partials/base.html' %}
```

Comment:
I removed partials/base.html because it didn't really seem to serve a purpose and having an additional "base.html" template seems confusing.